lib/Dialect/FIRRTL/Transforms/LowerXMR.cpp

```diff
@@ -488,7 +534,10 @@ class LowerXMRPass : public circt::firrtl::impl::LowerXMRBase<LowerXMRPass> {
               if (failed(resolveReference(op.getDest(), builder, ref, str)))
                 return failure();
 
-              Value xmr = builder.create<XMRRefOp>(destType, ref, str);
+              Value xmr =
+                  moduleStates.find(op->template getParentOfType<FModuleOp>())
```

Comment:
This `template` can cause problems with MSVCC. I'm checking if it works here: https://github.com/llvm/circt/actions/runs/12718725232

Reply:
This looks okay!